Tests/Core/CheckoutAPIClientTests.swift

```diff
@@ -9,6 +9,11 @@ class CheckoutAPIClientTests: XCTestCase {
         publicKey: "pk_test_6ff46046-30af-41d9-bf58-929022d2cd14",
         environment: .sandbox)
 
+    let everythingWithCorrectUserAgent: (_ request: URLRequest) -> Bool = { request in
+        let framesVersion = Bundle(for: CheckoutAPIClient.self).infoDictionary?["CFBundleShortVersionString"] as? String ?? "unknown"
```

Comment:
This line ideally in test should have been picked up from CheckoutAPIClient.swift line 18 and not duplicated. Can you some how use the headers set in CheckoutAPIClient.swift to include in your unit tests ?

Reply:
I was trying to not test the internals of `CheckoutAPIClient` here and instead test the headers on the `URLRequest`s, and I don't think making the tests dependent on `CheckoutAPIClient` internals would be ideal. I could hardcode the version string here but this would require changing with every version. What do you think is the best approach here?

Reply:
Ok makes sense for now, Probably we can refactor this bit as we go.